### ISSUE TITLE
server start time increase when switching from version 2.3.0 to 2.4.3

### ISSUE THREAD

**Issue by FrankSchiro** [openanalytics/shinyproxy#276]
I just switched from ShinyProxy 2.3.0 to 2.4.3

The startup time for the server went from maybe 3 seconds to 30 seconds.

I get this message as well: 

```
Multiple Spring Data modules found, entering strict repository configuration mode!
Bootstrapping Spring Data Redis repositories in DEFAULT mode
Finished Spring Data repository scanning in 71ms. Found 0 Redis repository interfaces
```

I'm wondering if this is an error of some kind? I never got this message at startup with 2.3.0. 

I also notice for version 2.3.0, it says ```Spring boot v2.0.0.RELEASE```, and for version 2.4.3, it says: ```Spring boot v2.3.4 release```

**Comment by LEDfan** [comment]
Hi @FrankSchiro 

I just did a quick test: I adapted ShinyProxy to exit the process as soon as ShinyProxy emits the `ContextRefreshedEvent` event. This event is emitted when ShinyProxy is completely stared up. Using the linux `time` command I then measured the time it takes for ShinyProxy to startup.

For ShinyProxy 2.3.0 this gives 8.14s and for ShinyProxy 2.4.3 11.36s. Indeed this is a bit slower, but since ShinyProxy gets some more features, some more initialization and checks must be performed at startup.

I don't recognize the startup of 30seconds. However, it may depends on your configuration. E.g. if you are using SAML, ShinyProxy has to fetch the metadata etc which may also take some time.

> Multiple Spring Data modules found, entering strict repository configuration mode!
> Bootstrapping Spring Data Redis repositories in DEFAULT mode
> Finished Spring Data repository scanning in 71ms. Found 0 Redis repository interfaces

This is normal behavior. This is just a message from the Spring framework that multiple modules are loaded which can be used for data handling. Nothing to worry about.

> I also notice for version 2.3.0, it says Spring boot v2.0.0.RELEASE, and for version 2.4.3, it says: Spring boot v2.3.4 release

In the latest ShinyProxy releases, the Spring framework was upgraded to 2.3.4, so this is expected behavior.
